draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -130,6 +130,9 @@ document are to be interpreted as described in RFC 2119 [@!RFC2119].
 This specification uses the terms "Holder", "Issuer", "Verifier", "Key Binding", and "Key Binding JWT" defined by
 [@!I-D.ietf-oauth-selective-disclosure-jwt].
 
+Consumer:
+: Applications using this specification are called Consumer. This typically includes Issuers, Verifiers, and Wallets.
```

Comment:
```suggestion
: Applications using the type metadata specified in (#type-metadata) are called Consumer. This typically includes Issuers, Verifiers, and Wallets.
```

Reply:
Thanks, I wanted to change this as well :)

Reply:
Thank you, yes, this needed the additional context.